pydm/widgets/shell_command.py

```diff
@@ -126,6 +129,29 @@ def showConfirmDialog(self, value: bool) -> None:
         if self._show_confirm_dialog != value:
             self._show_confirm_dialog = value
 
+    @Property(bool)
+    def runCmdsInFullShell(self) -> bool:
```

Comment:
Can probably spell out the full word commands for the new property just to match the other properties in this widget and the style of the widget property names in general